packages/expo-location/build/Location.types.d.ts

```diff
@@ -194,6 +194,12 @@ export type LocationTaskServiceOptions = {
      * Boolean value whether to destroy the foreground service if the app is killed.
      */
     killServiceOnDestroy?: boolean;
+    /**
+     * Boolean value whether to show the notification in the status bar. Defaults to false.
+     *
+     * @android
+     */
+    ongoing?: boolean;
```

Comment:
You edited this in the wrong place. 
You need to edit `Location.types.ts` inside the `/src` directory. Then you should run `yarn build` inside the `expo-location` directory.

To make sure everything is all right after you commit, but before you push you should run `et cp expo-location` and fix any errors if they pop up.